### ISSUE TITLE
Not able to transfert repository

### ISSUE THREAD

**Issue by Tcharl** [jhipster/jhipster-dependencies#1]
Hi all,

I'm not able to transfert the ownership of the [jhipster-parent](https://github.com/Tcharl/jhipster-parent) repo.

Can you fork and push?
In order to release it, can you generate a gpg (gpg --gen-key), then mvn release:prepare release:perform -Prelease?

Regards,
Charlie

**Comment by jdubois** [comment]
Yes I'll have a look (not sure when, but that's easy, I already do it for
jhipster/jhipster)

Le 30 sept. 2017 5:27 PM, "Charlie Mordant" <notifications@github.com> a
écrit :

**Comment by PierreBesson** [comment]
@tcharl I think you can't transfer ownership to JHipster as you are not member of the jhipster github org. I did it without problems in the past.

**Comment by jdubois** [comment]
I already had the issue: you can only transfer to a person, not an org. So basically you need to transfer to me and I'll transfer to the org.
